src/crypto/hsm/nxp/PersistentStorageOperationalKeystoreHSM.cpp

```diff
@@ -107,6 +107,8 @@ CHIP_ERROR PersistentStorageOperationalKeystoreHSM::NewOpKeypairForFabric(Fabric
     // Replace previous pending keypair, if any was previously allocated
     ResetPendingKey();
 
+    slotId = getEmpytSlotId();
```

Comment:
Why not just declare `slotId` here?  This is not C; you don't have to declare all your variables at the top of the function.